documentation/modules/ROOT/pages/configuration/filtering.adoc

```diff
@@ -40,62 +47,108 @@ transforms.filter.condition=value.op == 'u' && value.before.id == 2
+
+Typically, you add the language JAR to the directory that contains the {prodname} connector JAR files, which is the directory that is referenced in the `plugin.path` configuration property that is set for the Kafka Connect plugin. 
+If you install the script engine JAR to a different directory, you must add that directory to the list in the `plugin.path` property. 
+
 ====
 
-{prodname} binds four variables into the evaluation context:
+// Type: concept
+// ModuleID: variables-for-use-in-filter-expressions
+== Variables for use in filter expressions
+
+When you create expressions to specify filter conditions, you can use the variables that {prodname} binds into the evaluation context.
+Certain variables are bound into the evaluation context for the SMT.
+By binding variables , {prodname} enables the SMT to look up and interpret their values as it evaluates the conditions in an expression. 
+
+The following table lists the variables that {prodname} binds into the evaluation context for the filter SMT:
 
-* `key` - a key of the message
-* `value` - a value of the message
-* `keySchema` - the schema of the message key
-* `valueSchema` - the schema of the message value
-* `topic` - the name of the target topic
-* `headers` - the map of message headers keyed with header name and value composed of `schema` and `value` variables
+.Filter expression variables
+[options="header"]
+|=======================
+|Name |Description |Type
+|`key`   |A key of the message. |`org.apache.kafka.connect.data.Struct`
+|`value` |A value of the message. |`org.apache.kafka.connect.data.Struct`
+|`keySchema` |Schema of the message key.|`org.apache.kafka.connect.data.Schema`
+|`valueSchema`|Schema of the message value.| `org.apache.kafka.connect.data.Schema`
+|`topic`|Name of the target topic.| String
+|`headers`
+a|A Java map of message headers. The key field is the header name. 
+The `headers` variable exposes the following properties:
 
-The `key` and `value` are of type `org.apache.kafka.connect.data.Struct` and `keySchema` and `valueSchema` are variables of type `org.apache.kafka.connect.data.Schema`.
-The expression can invoke arbitrary methods on the variables and should evaluate into a boolean value that decides whether the message is removed `true` or kept.
-Expressions should be side-effect free, i.e. they should *not* modify the passed variables in any way.
+* `value` (of type `Object`) 
 
+* `schema` (of type `org.apache.kafka.connect.data.Schema`)
+
+| `java.util.Map<String, io.debezium.transforms.scripting.RecordHeader>`
+|=======================
+
+An expression can invoke arbitrary methods on its variables. 
+Expressions should resolve to a Boolean value that determines how the SMT dispositions the message.
+When the filter condition in an expression evaluates to `true`, the message is retained. 
+Whenthe filter condition evaluates to `false`, the message is removed.
```

Comment:
```suggestion
When the filter condition evaluates to `false`, the message is removed.
```

Reply:
I made this change as part of the latest commit.